### DISCUSSION TITLE
Cropping model image

### DISCUSSION THREAD

**Discussion by strstnts** [google/model-viewer#3467] [Q&A]
Hi! Is there a way to crop an image created through .toBlob() method?
![Capture](https://user-images.githubusercontent.com/15360316/168492897-0a9f1536-9752-43a6-bb49-afaa3d4341e7.PNG)

**Comment by elalish** [comment]
I would recommend `idealAspect: true`: https://modelviewer.dev/docs/index.html#entrydocs-loading-methods-toBlob

Also, `toBlob` is just a canvas screen shot, so to crop, you just need to set your element dimensions and `camera-orbit` to your desired view.
